src/slot.c

```diff
@@ -464,6 +465,7 @@ void node_list_get(char *dest)
 
 void slot_create_job(int type)
 {
+    incr_slot_update_counter();
```

Comment:
The other job is only called at exit, so I think there's no need to check
job type, to save some minor cpu

On Tue, Jul 5, 2016 at 11:45 PM doyoubi <notifications@github.com> wrote:

> In src/slot.c
> <https://github.com/eleme/corvus/pull/84#discussion_r69587033>:
>
> > @@ -464,6 +465,7 @@ void node_list_get(char *dest)
> >
> >  void slot_create_job(int type)
> >  {
> > +    incr_slot_update_counter();
>
> if (type == SLOT_UPDATE)
>     incr_slot_update_counter();
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eleme/corvus/pull/84/files/d67954be364fa0ea558bf7340664efac5a138337#r69587033>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABFB6Dva0u6PE3nSeRZCmWaJaraMT_M2ks5qSnwKgaJpZM4JEzIE>
> .
>